app/src/main/java/com/duckduckgo/app/browser/BrowserActivity.kt

```diff
@@ -106,6 +109,7 @@ import timber.log.Timber
 
 // open class so that we can test BrowserApplicationStateInfo
 @InjectWith(ActivityScope::class)
+@ContributeToActivityStarter(BrowserActivityWithParams::class)
```

Comment:
is this needed? can we get on without adding it? seems like the change to extract the ai chat webview shouldn’t need this to be added?

Reply:
Removed this, using `BrowserNav` instead.